charts/coturn/templates/configmap-coturn-conf-template.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "coturn.fullname" . }}
+  labels:
+    {{- include "coturn.selectorLabels" . | nindent 4 }}
+
+data:
+  coturn.conf.template: |
+    ## disable (d)tls control plane; don't permit relaying tcp connections.
+    no-tls
+    no-dtls
+    no-tcp-relay
+
+    ## don't turn on coturn's cli.
+    no-cli
+
+    ## authentication setup
+    # FUTUREWORK: enable support for multiple secrets?
+    zrest
+    static-auth-secret=__COTURN_SECRET__
+
+    ## turn, stun.
+    listening-ip=__COTURN_EXT_IP__
+    listening-port={{ .Values.coturnTurnListenPort }}
+    max-allocate-lifetime=3600
+    relay-ip=__COTURN_EXT_IP__
+    realm=dummy.io
+    no-stun-backward-compatibility
+    secure-stun
+    no-rfc5780
+
+    ## prometheus metrics
+    prometheus-ip=__COTURN_POD_IP__
+    prometheus-port={{ .Values.coturnMetricsListenPort }}
+
+    ## logs
+    log-file=stdout
+    verbose
+
+    ## access control settings.
+    # note that this does not include rfc1918 ipv4 or ipv6 ula address
+    # space. these ranges may be valid client ip addresses in some private
+    # network environments. however, the other ranges listed here are intended
+    # for special use and should not be used.
```

Comment:
If this is a list of "well known" ranges, can you link to the respective RFCs or other documents where those IP ranges are listed in a comment?

Reply:
I've amended this comment with a link to the appropriate IANA registries which list all the special-purpose addresses.